### PR TITLE
build: larger cgo stack

### PR DESCRIPTION
**Pull request by docmerlin**
This increases the cgo stack size of builds.

### CODE REVIEW

File: build.py [hunk 1]
```diff
@@ -535,24 +535,13 @@ def build(version=None,
...
-                build_command += "-ldflags=\"-s -X main.version={} -X main.branch={} -X main.commit={} -X main.platform=OSS\" ".format(version,
-                                                                                                                  get_current_branch(),
-                                                                                                                  get_current_commit())
-            else:
-                build_command += "-ldflags=\"-X main.version={} -X main.branch={} -X main.commit={} -X main.platform=OSS\" ".format(version,
+        build_command += "-ldflags=\""
+        if static:
+            build_command +="-s "
+
+        build_command += "-extldflags \\\"-fno-PIC -Wl,-z,stack-size=8388608\\\"  -X main.version={} -X main.branch={} -X main.commit={} -X main.platform=OSS\" ".format(version,
```
Comment (lesam) [inline build.py:544]: This might be clearer with triple-quotes `"""python triple quotes for my raw string so I don't have to "escape" backslash \ ..."""`

**Comment by lesam** [comment]
Did we validate that this is fixing the problem we saw with segfaults?

**Comment by docmerlin** [comment]
We have not, I was unable to reproduce them however.

File: build.py (hunk 1 above)
Comment (docmerlin) [reply build.py:544]: That is incorrect, in python to make it not need to escape you can use a an `r` before the string, example an `r'\'` is a slash.  Triple quotes just allow for multi-line strings, but they still have to be escaped.

Comment (lesam) [reply build.py]: Ah, TIL - my python is rusty. Thanks!

**Review by lesam — APPROVED** [review]

**Pull request merged**